src/libraries/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.cs

```diff
@@ -36,6 +36,13 @@ public static void WriteIndentation(Span<byte> buffer, int indent, byte indentBy
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why does this need to be explicitly inlined? It doesn't appear to be called in any of the hot paths.

Reply:
On closer inspection, I see that this just replicates other validation methods in the same file. Seems questionable, but keeping consistency would be preferable.

Reply:
I just copied what all the other validation methods were already doing.